examples/benchmark.py

```diff
@@ -31,140 +124,223 @@ def downloadAmberSuite():
         else:
             ff = app.ForceField('amoeba2009.xml', 'amoeba2009_gk.xml')
             pdb = app.PDBFile('5dfr_minimized.pdb')
-            system = ff.createSystem(pdb.topology, nonbondedMethod=app.NoCutoff, constraints=constraints, mutualInducedTargetEpsilon=epsilon, polarization=options.polarization)
+            system = ff.createSystem(pdb.topology, nonbondedMethod=app.NoCutoff, constraints=constraints, mutualInducedTargetEpsilon=epsilon, polarization=polarization)
         for f in system.getForces():
             if isinstance(f, mm.AmoebaMultipoleForce) or isinstance(f, mm.AmoebaVdwForce) or isinstance(f, mm.AmoebaGeneralizedKirkwoodForce) or isinstance(f, mm.AmoebaWcaDispersionForce):
                 f.setForceGroup(1)
-        dt = 0.002*unit.picoseconds
-        if options.ensemble == 'NVE':
-            integ = mm.MTSIntegrator(dt, [(0,2), (1,1)])
-        else:
-            integ = mm.MTSLangevinIntegrator(temperature, friction, dt, [(0,2), (1,1)])
         positions = pdb.positions
     elif amber:
         dirname = downloadAmberSuite()
-        names = {'amber20-dhfr':'JAC',  'amber20-factorix':'FactorIX', 'amber20-cellulose':'Cellulose', 'amber20-stmv':'STMV'}
+        names = {'amber20-dhfr':'JAC', 'amber20-factorix':'FactorIX', 'amber20-cellulose':'Cellulose', 'amber20-stmv':'STMV'}
         fileName = names[testName]
         prmtop = app.AmberPrmtopFile(os.path.join(dirname, f'PME/Topologies/{fileName}.prmtop'))
         inpcrd = app.AmberInpcrdFile(os.path.join(dirname, f'PME/Coordinates/{fileName}.inpcrd'))
         positions = inpcrd.positions
-        dt = 0.004*unit.picoseconds
         method = app.PME
-        cutoff = options.cutoff
         constraints = app.HBonds
         hydrogenMass = 1.5*unit.amu
+        cutoff = pme_cutoff
         system = prmtop.createSystem(nonbondedMethod=method, nonbondedCutoff=cutoff, constraints=constraints)
-        if options.ensemble == 'NVE':
-            integ = mm.VerletIntegrator(dt)
-        else:
-            integ = mm.LangevinMiddleIntegrator(temperature, friction, dt)
+        if inpcrd.boxVectors is not None:
+            system.setDefaultPeriodicBoxVectors(*inpcrd.boxVectors)
     else:
         if apoa1:
             ff = app.ForceField('amber14/protein.ff14SB.xml', 'amber14/lipid17.xml', 'amber14/tip3p.xml')
             pdb = app.PDBFile('apoa1.pdb')
             if testName == 'apoa1pme':
                 method = app.PME
-                cutoff = options.cutoff
+                cutoff = pme_cutoff
             elif testName == 'apoa1ljpme':
                 method = app.LJPME
-                cutoff = options.cutoff
+                cutoff = pme_cutoff
             else:
+                # Reaction field uses hard-coded cutoff
                 method = app.CutoffPeriodic
-                cutoff = 1*unit.nanometers
+                cutoff = 1*unit.nanometers # JDC: Shouldn't this be larger for reaction field?
```

Comment:
1 nm is a common value to use with reaction field.

Reply:
Any idea where this number comes from? Is there at least some precedent we can cite?